pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1235,22 +1235,24 @@ private CompletableFuture<Optional<Topic>> createNonPersistentTopic(String topic
         CompletableFuture<Optional<Topic>> topicFuture = new CompletableFuture<>();
         topicFuture.exceptionally(t -> {
             pulsarStats.recordTopicLoadFailed();
-            return null;
+            return Optional.empty();
```

Comment:
Same as the other: This change is not necessary since the stage returned by exceptionally() is not being used